packages/ag-grid-theme/css/_odyssey-ag-theme-default-params.scss

```diff
@@ -0,0 +1,93 @@
+@import "~ag-grid-community/src/styles/ag-theme-base/sass/ag-theme-base-default-params";
+@import "./odyssey-ag-theme-font-vars";
+
+$odyssey-ag-theme-default-params: (
+  //grid-size: 8px,
```

Comment:
Do we need all this commented out code?

Reply:
@joshwooding removed them now to move this PR along - we could add them back in as & when needed